src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/moderation/AttributeModifierWrapper.java

```diff
@@ -0,0 +1,18 @@
+package pl.tomaszdziurko.jvm_bloggers.view.admin.moderation;
+
+import org.apache.wicket.AttributeModifier;
+import org.apache.wicket.behavior.AttributeAppender;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+
+/**
+ * @autor mszarlinski
+ */
+@Component
+public class AttributeModifierWrapper {
```

Comment:
I've made this component to avoid stubbing static API of `AttributeModifier` class from Wicket. I know it can be achieved with `GroovyMock` probably, but maybe this is a cleaner way.